.github/workflows/macos.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest]
+        os: [macos-10.15]
```

Comment:
is this the intention? can you move this bit into separate commit?

Reply:
It is to solve this problem:  https://github.com/libevent/libevent/runs/705558000?check_suite_focus=true

Reply:
The runner has been updated from 2.169.1 to 2.263.0 by the github actions team. This will invalidate the build cache. So we need to retry as you mentioned.